firmware/library/L2_Utilities/utility_test.cpp

```diff
@@ -0,0 +1,126 @@
+#include "L2_Utilities/utility.hpp"
+#include "L5_Testing/testing_frameworks.hpp"
+
+TEST_CASE("Testing Random Functions", "[random]")
+{
+    SECTION("SetRandomSeed Consistency")
+    {
+        utility::SetRandomSeed();
+        uint32_t randomtester1 = std::rand() % 100;
+        utility::SetRandomSeed();
```

Comment:
Just realized that you should have another set of conditional tests for SetRandomSeed where you actually give it a seed value and make sure that works as expected.